modules/ROOT/pages/managing-application-data-with-object-stores.adoc

```diff
@@ -31,9 +31,9 @@ Your application can also store custom integration data as well, within the limi
 To work with application data inside a Mule flow, use the Object Store connector. The connector provides operations to store, retrieve, list, and remove data from a Runtime Manager Object Store. The Object Store connector works with both Object Store v1 and Object Store v2.
 
 
-=== Store OAuth Data
+=== Retrieve OAuth Data
 
-For information on how to use the Object Store module to store and retrieve OAuth tokens, see the "Saving and Restoring State" section in the http://mulesoft.github.com/linkedin-connector/mule/linkedin-config.html#config[LinkedIn connector configuration] documentation for an example.
+For information on how to retrieve OAuth tokens, see xref:oauth-module::index.adoc[OAuth Module].
```

Comment:
see the ...

Reply:
As with connectors, we don't use articles with module names.